applications/konami/src/module/konami_move.erl

```diff
@@ -93,13 +93,14 @@ get_originate_req(Data, Call) ->
                                  ,{<<"can_call_self">>, 'true'}
                                 ], Data),
 
-    OwnerId = wh_json:get_value(<<"owner_id">>, Data),
     {SourceDeviceId, TargetCallId} =
         case source_leg_of_dtmf(SourceOfDTMF, Call) of
             'a' -> {whapps_call:authorizing_id(Call), whapps_call:other_leg_call_id(Call)};
             'b' -> {find_device_id_for_leg(SourceOfDTMF), whapps_call:call_id(Call)}
         end,
 
+    DeviceOwnerId = cf_attributes:owner_ids(SourceDeviceId, Call),
```

Comment:
@Dinkor cf_attributes:owner_ids belongs to callflow whapp. you are breaking the whapp abstraction here (konami may run in one server and callflow in another). can you find another way to do it ?